templates/TagCloud.php.twig

```diff
@@ -0,0 +1,6 @@
+{% extends 'base.html.twig' %}
+
+{% block content %}
+    <h3>The most popular searches</h3>
+    {{ include('blocks/_TagCloud.html.twig') }}
```

Comment:
use snake case for twig filenames (https://symfony.com/doc/current/templates.html#template-naming)